modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -3053,7 +3053,7 @@ definition {
 				key_column = "External Reference Code",
 				locator1 = "ObjectPortlet#ENTRY_COLUMN");
 
-			var externalReferenceCodeId = selenium.getText("//div[contains(@class,'dnd-tbody')]//*[contains(text(),'Entry Test')]/parent::div/following-sibling::div//a");
+			var externalReferenceCodeId = selenium.getText("//tr[td/span[contains(text(), 'Entry Test')]]/td[@aria-colindex='2']");
```

Comment:
Will this selector return an object definition external reference code? 

If so could you rename it to `objectDefinitionExternalReferenceCode`? If not, could you tell me what the return is 😅

Reply:
I don't remember @igor-franca, will check this and get back to you.
I just adjusted the locator to continue selecting what it was before 😅

Reply:
this is the entry ERC.